internal/consensus/replay_test.go

```diff
@@ -1162,14 +1166,16 @@ func stateAndStore(
 // mock block store
 
 type mockBlockStore struct {
-	cfg     *config.Config
-	params  types.ConsensusParams
-	chain   []*types.Block
-	commits []*types.Commit
-	base    int64
-	t       *testing.T
+	cfg        *config.Config
+	params     types.ConsensusParams
+	chain      []*types.Block
+	extCommits []*types.ExtendedCommit
```

Comment:
Similarly not convinced this couldn't have remained `commits`, but YMMV.

Reply:
Same as above.